### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by ysb33r**
## Features
- #347 Use docExtensions instead extensions, the latter is deprecated.
- #348 Use modules block as explicit versions are deprecated.
- #369 `Basedir` can be controlled via strategy

## Bugfixes
- #349 Make `asciidoctorRevealJs` work on Gradle 4.x.
- #371 `UNSAFE` is now the default safe mode.
- #366 `asciidoctorPdf`, `asciidoctorEpub` & `asciidoctorRevealJs` now has a default source directory of `src/docs/asciidoc`
- #368 `docinfo` usages requires `baseDirFollowsSourceDir` strategy.

## Other
- #354 Codenarc rules improved.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -286,6 +324,11 @@ All Asciidoctor tasks will have the following methods and properties:
 asciidoctorj:: a task extension which allows a task to extend of override global configuration for Asciidoctor tasks.
   This allow extensive flexibility. Any thing that can be configured in the global `asciidoctorj` extension can also be configured here.
 attributes:: A shortcut for `asciidoctorj.attributes`.
+baseDir:: Base directory for asciidoctor jail.
```
Comment (aalmiray) [inline README.adoc]: "jail" ?

File: README.adoc [hunk 2]
```diff
@@ -286,6 +324,11 @@ All Asciidoctor tasks will have the following methods and properties:
...
+  The base directory will be the project directory, but default, but can be set to any other directory.
+baseDirFollowsSourceDir:: The base directory should be the same as the source directory even if the source diretory is located within an intermediate working directory.
```
Comment (aalmiray) [inline README.adoc]: s/diretory/directory/

File: README.adoc (hunk 1 above)
Comment (ysb33r) [reply README.adoc]: Maybe need a better word, but basically Asciidoctor cannot access anything outside a jailed base directory unless the safety mode is not `UNSAFE`.

Comment (ysb33r) [reply README.adoc]: Updated the text.

File: README.adoc (hunk 2 above)
Comment (ysb33r) [reply README.adoc]: Thanks!

**Comment by ysb33r** [comment]
If you're happy you can start the release process.

**Pull request merged**